scripts/ci-publish.sh

```diff
@@ -8,3 +8,7 @@ npm config set //registry.npmjs.org/:_authToken ${NPM_TOKEN}
 
 # Publish dev build via lerna
 npm run-script lerna-ci-publish
+
+echo Publishing demo app
+cd package/app
+npm run-script deploy-demo-app
```

Comment:
I think this is the right place to push changes when a new packages is published?